lib/services/nativescript-project-plugins-service.ts

```diff
@@ -358,6 +343,68 @@ export class NativeScriptProjectPluginsService implements IPluginsService {
 		}).future<IPlugin>()();
 	}
 
+	private getDataForLocalPlugin(packageName: string, pathToPlugin?: string): IFuture<IPlugin> {
+		return ((): IPlugin => {
+			let plugin: IPlugin;
+			if(!!pathToPlugin.match(/^file:/)) {
+				pathToPlugin = pathToPlugin.replace("file:", "");
+			}
+
+			if(this.checkIsValidLocalPlugin(pathToPlugin).wait()) {
+				let fullPath = path.resolve(pathToPlugin);
+				let packageJsonContent = this.$fs.readText(path.join(fullPath, this.$projectConstants.PACKAGE_JSON_NAME)).wait();
+				plugin = this.constructNativeScriptPluginData(packageJsonContent).wait();
+			}
+
+			return plugin;
```

Comment:
This returns implicitly `undefined`. Should we return an explicit `null` instead?